pupil_src/shared_modules/math_helper/transformations.py

```diff
@@ -1980,6 +1983,33 @@ def _import_module(name, package=None, warn=True, prefix="_py_", ignore="_"):
         return True
 
 
+def find_rigid_transform(A, B):
+    # we expect the shape to be of length 2
+    assert len(A.shape) == len(B.shape) == 2
+    assert A.shape[0] == B.shape[0]
+
+    centroid_A = np.mean(A, axis=0)
+    centroid_B = np.mean(B, axis=0)
+
+    # centre the points
```

Comment:
Please iterate over comments and variable names in this function. Some comments are not necessary. Please also solve the todo that is mentioned in one comment.

Reply:
See above.